src/app/app-config.model.ts

```diff
@@ -37,6 +37,7 @@ export interface FeaturesRuntimeConfig {
 
 export interface AppUIConfig {
     ingestUploadFileLimitMb: number;
+    semanticSearchScore?: number;
```

Comment:
It's not simply a score, but a "treshold score".

Reply:
Renamed